wasp-ai/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "singleQuote": false,
+  "trailingComma": "es5",
+  "printWidth": 120
```

Comment:
Ha, I read in their docs that this is not the same as actual width of how long the lines should be and that they advise leaving this at 80, even if we are going with lines of 100 or 120. I was just reading this yesterday. But I do have to admit I find it quite annoying, it is overly aggressive with breaking those lines.
I would however advise going with 100, I found that to be best number so far -> it is short enough to still mostly fit into smaller editors like this one on github, but is still not as short as 80 which is just annoying.